src/main/java/it/gov/pagopa/controller/consent/PortalConsentControllerImpl.java

```diff
@@ -26,14 +22,17 @@ public class PortalConsentControllerImpl implements PortalConsentController {
     private String linkTc;
 
     @Override
-    //@ResponseStatus(HttpStatus.OK)
-    public ResponseEntity<PortalConsentDTO> getPortalConsent(@RequestHeader(value = "Content-Language") String contentLanguage) throws JsonProcessingException {
-        return null;
+    public ResponseEntity<PortalConsentDTO> getPortalConsent(String userId) throws JsonProcessingException {
+        PortalConsentDTO consentDTO = portalConsentService.get(userId);
+
+        return consentDTO != null
+                ? ResponseEntity.ok(consentDTO)
+                : ResponseEntity.notFound().build();
     }
 
     @Override
-    //@ResponseStatus(HttpStatus.OK)
-    public ResponseEntity<PortalConsentDTO> savePortalConsent(@RequestHeader(value = "Content-Language") String contentLanguage) throws JsonProcessingException{
-        return null;
+    public ResponseEntity<Void> savePortalConsent(String userId, PortalConsentDTO consent) throws JsonProcessingException{
+        portalConsentService.save(userId, consent);
+        return ResponseEntity.ok().build();
     }
```

Comment:
it should be possible to change it into this simpler solution
```suggestion
    public void savePortalConsent(String userId, PortalConsentDTO consent) throws JsonProcessingException{
        portalConsentService.save(userId, consent);
    }
```